newIDE/app/src/GameDashboard/GamesList.js

```diff
@@ -139,115 +373,261 @@ const GamesList = ({
-            size="small"
-          >
-            <ChevronArrowLeft />
-          </IconButton>
-          <Text
-            noMargin
-            style={{
-              opacity: searchText ? 0.6 : 1,
-              fontVariantNumeric: 'tabular-nums',
-            }}
-          >
-            {searchText ? 1 : currentPage + 1}
-          </Text>
-          <IconButton
-            tooltip={t`Next page`}
-            onClick={() => setCurrentPage(currentPage => currentPage + 1)}
-            disabled={
-              !!searchText || (currentPage + 1) * pageSize >= games.length
-            }
-            size="small"
-          >
-            <ChevronArrowRight />
-          </IconButton>
-        </Line>
-      </ResponsiveLineStackLayout>
-      {displayedGames.length > 0 ? (
-        displayedGames.map(game => (
-          <GameCard
-            storageProviders={storageProviders}
-            key={game.id}
-            isCurrentGame={!!projectUuid && game.id === projectUuid}
-            game={game}
-            onOpenGameManager={() => {
-              onOpenGameId(game.id);
-            }}
-            onOpenProject={onOpenProject}
-          />
-        ))
-      ) : !!searchText ? (
-        <Column expand noMargin>
-          <Paper
-            variant="outlined"
-            background="dark"
-            style={styles.noGameMessageContainer}
-          >
-            <BackgroundText>
-              <Trans>No game matching your search.</Trans>
-            </BackgroundText>
-          </Paper>
-        </Column>
-      ) : null}
-    </ColumnStackLayout>
+        );
+        setLastModifiedInfoByProjectId(_lastModifiedInfoByProjectId);
+      };
+
+      updateModificationInfoByProjectId();
+    },
+    [cloudProjects, profile]
+  );
+
+  const shouldShowOpenProject =
+    canOpen &&
+    // Only show on large screens.
+    !isMobile &&
+    // Only show on desktop as otherwise, cloud projects are the only ones that can be opened.
+    isDesktop;
+
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <ColumnStackLayout noMargin>
+          <Line noMargin justifyContent="space-between">
+            <LineStackLayout noMargin alignItems="center">
+              <Text size="section-title" noMargin>
+                <Trans>Games</Trans>
+              </Text>
+              {allDashboardItems.length > 0 && (
+                <IconButton
+                  size="small"
+                  onClick={refreshGamesList}
+                  disabled={isRefreshing}
+                  tooltip={t`Refresh games`}
+                >
+                  <div style={styles.refreshIconContainer}>
+                    <Refresh fontSize="inherit" />
+                  </div>
+                </IconButton>
+              )}
+            </LineStackLayout>
+            <LineStackLayout noMargin alignItems="center">
+              <RaisedButton
+                primary
+                label={
+                  isMobile ? (
```

Comment:
```suggestion
                  isMobile && !isLandscape ? (
```
(Not sure)

Reply:
I'll do a full pass on landscape to check